oommfc/tests/test_oommf.py

```diff
@@ -122,7 +122,8 @@ def test_get_oommf_runner():
 
 def test_get_oommf_runner():
     # This is a shorter version of the previous test for testing on host.
-    oommf_runner = oo.get_oommf_runner(use_cache=False)
+    oc.runner.autoselect_runner()
```

Comment:
This new test `test_get_oommf_runner` checks that some OOMMF can be found (on the testing system).

There is plenty of code marked with 
```@pytest.mark.skip(reason='Temporary')``` - is the intention to re-activate some/all of that?

We can have an in-person review/discussion if that helps.

Reply:
I don't know. It has already been deactivated. We need to ask @marijanbeg and resolve this (best in-person).